src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -1258,7 +1258,7 @@ private static function mentionCallback($match)
 			return $match[0];
 		}
 
-		return '@[url=' . $data['url'] . ']' . $data['nick'] . '[/url]';
+		return '[url=' . $data['url'] . ']@' . $data['nick'] . '[/url]';
```

Comment:
In Pleroma and Mastodon the `@` is outside the link, it is just some CSS tricks that it looks like it is part of the link.

Reply:
Nope, see Mastodon HTML code for a mention:
```html
<a href="https://mastodon.xyz/@lightone" class="u-url mention status-link" title="lightone@mastodon.xyz" target="_blank" rel="noopener noreferrer">
    @<span>lightone</span>
</a>
```

Reply:
There are also differences between Pleroma and Mastodon. But we can see if there are side effects or not.